kinds/accesspolicy/access_policy_kind.cue

```diff
@@ -0,0 +1,46 @@
+package kind
+
+name:              "AccessPolicy"
+maturity:          "merged"
+description:       "Access rules for a scope+role.  NOTE there is a unique constraint on role+scope"
+pluralName:        "AccessPolicies"
+machineName:       "accesspolicy"
+pluralMachineName: "accesspolicies"
+
+lineage: seqs: [
+	{
+		schemas: [
+			// v0.0
+			{
+				spec: {
+					// The scope where these policies should apply
+					scope: #ResourceID
```

Comment:
@IevaVasiljeva -- following up on comments from a few weeks ago in https://github.com/grafana/grafana/pull/66714/files#r1171594880 -- the goal here is to mirror the existing information.  The scope would match what exists in the existing scope.

This will work naturally for things that are real "kinds" dashboard/folder/alert/etc, but I *think* current RBAC is using scope to limit access beyond just resources.  Are there good examples of that?